examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,587 @@
+{
```

Comment:
<p>super duper nitpick: <code>it's</code> if you mean "after it has run". Otherwise this means "after the task's run"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>